docs/070-breaking-changes.rst

```diff
@@ -8,13 +8,113 @@ affected code.
 For the full list check
 `the release changelog <https://github.com/ethereum/solidity/releases/tag/v0.7.0>`_.
 
+
+Silent Changes of the Semantics
+===============================
+
+ * Exponentiation and shifts of literals by non-literals (e.g. ``1 << x`` or ``2 ** x``)
+   will always use either the type ``uint256`` (for non-negative literals) or
+   ``int256`` (for negative literals) to perform the operation.
+   Previously, the operation was performed in the type of the shift amount / the
+   exponent which can be misleading.
+
+
+Changes to the Syntax
```

Comment:
```suggestion
Semantic and Syntactic Changes
```
If we rename it to this we would be consistent with the 0.6 doc, in case it fits.